nilearn/decoding/space_net.py

```diff
@@ -1329,34 +1328,6 @@ def score(self, X, y):
         check_is_fitted(self)
         return accuracy_score(y, self.predict(X))
 
-    def _more_tags(self):
-        """Return estimator tags.
-
-        TODO remove when bumping sklearn_version > 1.5
-        """
-        return self.__sklearn_tags__()
```

Comment:
If  `_more_tags` is not necessary here, may be it is not necessary above as well?